dbms/tests/queries/0_stateless/01073_crlf_in_output_csv_format.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Ok, but it's better to make `.sql` test with `.reference` file containing expected output